environments/vagrant-example/activate

```diff
@@ -1,6 +1,9 @@
 export APPLIANCES_ENVIRONMENT_ROOT=$(dirname $(realpath ${BASH_SOURCE[0]:-${(%):-%x}}))
 echo "Setting APPLIANCES_ENVIRONMENT_ROOT to $APPLIANCES_ENVIRONMENT_ROOT"
 
+APPLIANCES_ENVIRONMENT_NAME=$(basename $APPLIANCES_ENVIRONMENT_ROOT)
```

Comment:
nit: unrelated to change

Reply:
True, brings vagrant into line with cookiecutter though.